editoast/src/views/timetable.rs

```diff
@@ -442,23 +446,84 @@ async fn conflicts(
     )
     .await?;
 
+    let mut paced_trains_ts = vec![];
+    for paced_train in paced_trains.clone() {
+        if paced_train.step.num_seconds() == 0 {
```

Comment:
Do we even allow zero-stepped paced trains? :o 
What's the reason behind that?

Reply:
It should be forbidden. This field should be strictly positive.